docs/source/scripts/plot_contour.py

```diff
@@ -0,0 +1,23 @@
+import optuna
+import plotly
+
+
+def main():
+    def objective(trial):
+        x = trial.suggest_uniform('x', -100, 100)
+        y = trial.suggest_categorical('y', [-1, 0, 1])
+        return x ** 2 + y
+
+    sampler = optuna.samplers.TPESampler(seed=10)
+    study = optuna.create_study(sampler=sampler)
+    study.optimize(objective, n_trials=10)
+
+    fig = optuna.visualization.plot_contour(study, params=['x', 'y'])
+    fig_div = plotly.offline.plot(fig, output_type='div', include_plotlyjs=False, auto_open=False)
+
+    with open('../plotly_figures/plot_contour.html', 'w') as f:
```

Comment:
A detail but how about creating the `plotly_figures` directory if it doesn't exist, instead of raising an error. It's acceptable "overengineering" I think in favor of usability.

Reply:
You're right. I'll fix this.